config/initializers/git_revision.rb

```diff
@@ -1,8 +1,6 @@
 module AppInfo
   if Rails.env.production?
-    #TODO: git SHA available once devops PR #271 is merged:
-    #https://github.com/department-of-veterans-affairs/devops/pull/271
-    GIT_REVISION = "unknown"
+    GIT_REVISION = ENV["APP_GIT_SHA"]
```

Comment:
instead of putting this in a if statement (which will make test coverage go down), what if you just did:

```ruby
GIT_REVISION = ENV["APP_GIT_SHA"] || `git rev-parse HEAD`.chomp
```

I realize it's not 100% the same thing as what you have, but might suffice. There is nothing wrong with having conditional logic in initializers, but if we do we should probably also have specs for it.